lib/mondrian/olap/cube.rb

```diff
@@ -25,6 +25,7 @@ def self.get(connection, name)
       def initialize(connection, raw_cube)
         @connection = connection
         @raw_cube = raw_cube
+        @cache_control = CacheControl.new(@connection, @raw_cube)
```

Comment:
Better `CacheControl.new(@connection, self)` as `raw_cube` can be accessed via a method.